### ISSUE TITLE
Animate Acid/Fire

### ISSUE THREAD

**Issue by wherfz** [mlenser/kryx-rpg-issues#375]
Acid Form. The acid elemental can move through a space as narrow as 1 inch wide without squeezing. A creature that touches the acid elemental or hits it with a melee attack while within 5 feet of it takes 8 (2d4 + 3) acid damage. In addition, the acid elemental can enter a hostile creature’s space and stop there. **When another creature starts its turn in the elemental**, that creature takes 8 (2d4 + 3) acid damage and starts chemically burning; until someone takes an action to stop the burning, the creature takes 8 (2d4 + 3) acid damage at the start of each of its turns.

Instead of " The first time it enters a creature’s space on a turn, "
Otherwise, this spell can really out damage AoE spells like searing bolts/burning hands just by walking the elemental through everyone's spaces.

**Comment by itamarcu** [comment]
The current wording -
> The first time it enters a creature’s space on a turn
implies to me that it can only apply this special damage to a single creature during the elemental's turn. It's a bit ambiguous, though.

**Comment by mlenser** [comment]
The wording is structure to be exactly as it is in RAW:

https://www.dndbeyond.com/monsters/fire-elemental
> In addition, the elemental can enter a hostile creature's space and stop there. The first time it enters a creature's space on a turn, that creature takes 5 (1d10) fire damage and catches fire; until someone takes an action to douse the fire, the creature takes 5 (1d10) fire damage at the start of each of its turns.

**Comment by itamarcu** [comment]
Looking online...

- https://www.enworld.org/threads/how-does-a-fire-elementals-fire-form-work-in-5e.474673/
- https://rpg.stackexchange.com/questions/97577/can-a-fire-elemental-deal-damage-to-multiple-creatures-in-the-same-turn-by-movin

The consensus is that it *definitely can* hurt multiple creatures in a single turn, by disengaging and moving through up to 5 of them (or dashing and moving through 10 and taking opportunity attacks).

**Comment by itamarcu** [comment]
This definitely sounds too strong to me, so I suggest rewording it to "Once during its turn, when the elemental enters a creature's space, that...".

**Comment by mlenser** [comment]
We'll need to reword fire and acid elementals as well. To either the end turn wording or the wording you suggest. The end turn wording is more thematic.

**Comment by mlenser** [comment]
> by disengaging and moving through up to 5 of them (or dashing and moving through 10 and taking opportunity attacks).

Some notes:
- Moving through creatures treats them as difficult terrain, so a 30 foot speed creature could move through 3 creatures, max (all would have to be adjacent). Most commonly 2 creatures.
- Dashing would get you more creatures, 6 with perfect positioning. More likely 4 max. And you'd take an opportunity attack from each.

I don't think this is as good as it appears on paper.

**Comment by DalenWBrauner** [comment]
It might be worthwhile to include that explicitly.
> In addition, the acid elemental can enter a hostile creature’s space and continue moving, treating them as difficult terrain.

While it makes sense that normal creatures use that difficult terrain rule, I could see it being unclear whether an elemental that can squeeze through 1-inch gaps and has a specific mechanic for passing through creatures is also affected by it. This clears that up.

**Comment by DalenWBrauner** [comment]
Could also be worded
> treating them as difficult terrain as normal

If you want to avoid any implications of 'stacking' difficult terrain, but I'm not sure that's necessary (or how difficult terrain even works).

**Comment by mlenser** [comment]
> It might be worthwhile to include that explicitly.

It's part of the default rules:

> The space of another creature, whether hostile or not, also counts as difficult terrain.

________________________

> While it makes sense that normal creatures use that difficult terrain rule, I could see it being unclear whether an elemental that can squeeze through 1-inch gaps and has a specific mechanic

The Fire Elemental has no such wording that allows it to bypass the normal rules about other creatures being difficult terrain. Therefore it is unable to bypass that normal rule.

I can't possibly account for all situations where players impose their own feelings on rules.

**Comment by itamarcu** [comment]
> 30 foot speed

The elementals have a 50 foot speed - that's why I said 5 or 10 creatures (optimally).

**Comment by mlenser** [comment]
A Large fire elemental attacks twice for 2d6 (7) + 5 + 1d10 (5.5), or 17.5 each, or 35 damage for both attacks. Multiplied by 0.6 to account for attack and we're at around 21 DPR.
Moving through 5 creatures is 5*5.5 = ~27.5

So it has to either disengage or provoke a ton of opportunity attacks. Seems fairly ok, besides the burning part.

**Comment by mlenser** [comment]
Reduced burning from 3 to 2 on the huge fire elemental and from 2 to 1 on the medium fire elemental

**Comment by itamarcu** [comment]
Also note that the Acid elemental basically inflicts a primitive version of burning - I think it'd be nice if they were aligned somehow.

**Comment by DalenWBrauner** [comment]
> basically inflicts a primitive version of burning - I think it'd be nice if they were aligned somehow.

Whether or not Acid spells & creatures should inflict the Burning condition or a new, more acidic condition (or a different solution entirely) is probably worth discussing in its own issue. It would have other implications (and could potentially give the Acid theme more flavor).

**Comment by mlenser** [comment]
Acid and Fire Elementals have had their form changed so they burn creatures whose space they stay in
